docker/develop/aarch64/Dockerfile

```diff
@@ -0,0 +1,202 @@
+FROM ubuntu:16.04
+
+# number of concurrent threads during build
+# usage: docker build --build-arg PARALLELISM=8 -t name/name .
+ARG PARALLELISM=1
+
+ENV IROHA_HOME /opt/iroha
+ENV IROHA_BUILD /opt/iroha/build
+
+RUN apt-get update; \
+    apt-get -y upgrade; \
+    apt-get -y --no-install-recommends install apt-utils software-properties-common; \
+    apt-get -y clean
+
+# add git repository
+RUN add-apt-repository -y ppa:git-core/ppa; \
+    apt-get update
+
+RUN apt-get -y --no-install-recommends install build-essential python-software-properties \
+        automake libtool \
+        # dev dependencies
+        libssl-dev zlib1g-dev libc6-dbg golang \
+        # CircleCI dependencies
+        git ssh tar gzip ca-certificates python3 python3-pip python3-setuptools \
+        # other
+        wget curl file unzip gdb iputils-ping vim ccache \
+        gcovr cppcheck doxygen graphviz graphviz-dev; \
+    apt-get -y clean
+
+RUN pip3 install --upgrade pip; \
+    pip3 install flask requests
+
+# install cmake 3.5.1
+RUN git clone https://gitlab.kitware.com/cmake/cmake.git /tmp/cmake; \
+    (cd /tmp/cmake ; git checkout 64130a7e793483e24c1d68bdd234f81d5edb2d51); \
+    (cd /tmp/cmake ; /tmp/cmake/bootstrap --parallel=${PARALLELISM} --enable-ccache); \
+    make -j${PARALLELISM} -C /tmp/cmake; \
+    make -C /tmp/cmake install; \
+    rm -rf /tmp/cmake
+
+# install boost 1.65.1
+RUN git clone https://github.com/boostorg/boost /tmp/boost; \
+    (cd /tmp/boost ; git checkout 436ad1dfcfc7e0246141beddd11c8a4e9c10b146); \
+    (cd /tmp/boost ; git submodule init); \
+    (cd /tmp/boost ; git submodule update --recursive -j ${PARALLELISM}); \
```

Comment:
you need to fetch changes from https://github.com/hyperledger/iroha/blob/develop/docker/develop/Dockerfile

new git does not have an option `-j` and this may fail. 

do update for all dockerfiles

Reply:
I would recommend to wait for https://github.com/hyperledger/iroha/pull/1069